dg/lib/cloud_pak_for_data/cpd_4_0_0/types/cloud_pak_for_data_license.py

```diff
@@ -0,0 +1,22 @@
+#  Copyright 2021 IBM Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from enum import Enum, auto
+
+
+class CloudPakForDataLicense(Enum):
```

Comment:
Is this really the CPD license or the Db2 license? The types sound a lot like Db2 license types.

Reply:
Actually, this class should be named `CloudPakForDataServiceLicense` as it contains all license names I encountered when integrating the IBM Cloud Pak for Data services.

Examples:
- **Db2**: Advanced, Community, Standard
- **Db2 Data Gate**: Enterprise, Standard

Reply:
Sounds good to me.